react-components/package.json

```diff
@@ -48,6 +49,7 @@
     "@cognite/cogs.js": "^10.36.0",
     "@cognite/reveal": "^4.23.2",
     "@cognite/sdk": "^9.13.0",
+    "@cognite/signals": "^0.0.4",
```

Comment:
Remove this. It is already defined in "dependencies". 

The difference is that devDependencies are only needed for building/testing or other things that is not the actual source code.